python-requirements.txt

```diff
@@ -3,6 +3,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Keep sorted.
+anytree
```

Comment:
Can you add a comment that this dependency is used by the vendored-in  sw/host/vendor/google_verible_verilog_syntax_py code? (To make it easier to remove it once this functionality is available as Python package.)

Reply:
Done